src/libwalletqt/Wallet.cpp

```diff
@@ -458,6 +512,19 @@ AddressBookModel *Wallet::addressBookModel() const
     return m_addressBookModel;
 }
 
+Subaddress *Wallet::subaddress() const
+{
+    return m_subaddress;
+}
```

Comment:
I think I've already mentioned this in a previous PR, but I don't like this system of const functions returning non const pointers to their data. It can be kept as is though, not my repo.